packages/ts/react-crud/src/autocrud.obj.css

```diff
@@ -46,10 +40,19 @@
 
 .auto-crud .auto-form {
   width: 40%;
-  border: solid 1px var(--lumo-contrast-20pct);
-  border-left-width: 0;
-  box-shadow: var(--lumo-box-shadow-s);
+  min-width: 300px;
+}
+
+/* Move box shadow and required z-index modification into pseudo-element
+   as it otherwise messes with the drag handle of the split layout */
+.auto-crud .auto-form::before {
```

Comment:
Adding a z-index to the form caused the splitter handle to move to the left. This seems to work as an alternative for applying the box shadow.